### PR TITLE
BuzzFeed Presence

### PR DESCRIPTION
**Pull request by Lanson2232**

### CODE REVIEW

**Comment by frozaum** [comment]
Please provide us a screenshot of the presence, so we can review it. @Lanson2232

**Comment by Lanson2232** [comment]
**HomePage**:
Homepage


**Trending Page**:
trending

**Quiz Page**:
quizzes

**Search Page**:
Seach

File: BuzzFeed/dist/presence.js [hunk 1]
```diff
@@ -68,17 +68,17 @@ presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
...
         data.details = "Searching:"
         data.state = document.title;
         data.smallImageKey = "logo";
     } else if (user && user.textContent != "") {
-        data.details = "Shows to User Profile:"
+        data.details = "looks to a user Profile:"
         data.state = user.textContent.trim()
     } else {
-        data.details = "Shows to a Page:"
+        data.details = "looks to a Page:"
```
Comment (Bas950) [inline BuzzFeed/dist/presence.js]: Dont you mean:
"Viewing page:"

Comment (Lanson2232) [reply BuzzFeed/dist/presence.js]: Aaa yes Sorry My english is a2. Thats Why my English is bad

**Pull request merged**